docs/user/tutorials/01-50-expose-and-secure-a-workload/v2alpha1/01-53-expose-and-secure-workload-ext-auth.md

```diff
@@ -0,0 +1,108 @@
+
+# Expose and Secure a Workload with ExtAuth
+
+This tutorial shows how to expose and secure Services using APIGateway Controller. The Controller reacts to an instance of the 
+APIRule custom resource (CR) and creates an Istio [VirtualService](https://istio.io/latest/docs/reference/config/networking/virtual-service/), 
+[Authorization Policy](https://istio.io/latest/docs/reference/config/security/authorization-policy/) with action type `CUSTOM`.
+
+In this tutorial we will use a [oauth2-proxy](https://oauth2-proxy.github.io/oauth2-proxy/) with a OAuth2.0 complaint authorization server supporting oidc discovery 
+to secure a workload using OAuth2.0 Client Credentials flow.
+
+## Prerequisites
+
+* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
+* [Set up your custom domain](../../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead.
+* [Obtain a JSON Web Token (JWT)](../01-51-get-jwt.md).
+
+## Steps
+
+### Expose and Secure Your Workload
+
+#### **kubectl**
+
+1. Depending on whether you use your custom domain or a Kyma domain, export the necessary values as environment variables:
+
+<!-- tabs:start -->
+#### **Custom Domain**
+
+```bash
+export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+export GATEWAY=$NAMESPACE/httpbin-gateway
+```
+#### **Kyma Domain**
+
+```bash
+export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
+export GATEWAY=kyma-system/kyma-gateway
+```
+<!-- tabs:end -->
+
+2. Deploy `oauth2-proxy` with configuration for your authorization server. This tutorial uses [oauth2-proxy helm chart](https://github.com/oauth2-proxy/manifests) 
```

Comment:
Why do we add the whole Helm chart to the repo, when we have it [much simpler in Istio documentation](https://github.com/kyma-project/istio/blob/main/docs/user/tutorials/01-10-external-authorization-provider.md#install-and-configure-oauth2-proxy)?

Reply:
Did not see this one, will change that as well